sources/packages/backend/libs/services/src/queue/model/queue.model.ts

```diff
@@ -3,4 +3,6 @@ import { BullModuleOptions } from "@nestjs/bull";
 export interface QueueModel extends Omit<BullModuleOptions, "redis"> {
   name: string;
   dashboardReadonly?: boolean;
+  isActive: boolean;
```

Comment:
Are we not keeping this as optional?

Reply:
I do not see a reason why this would be optional.